portal-impl/src/com/liferay/portlet/internal/PortletURLFactoryImpl.java

```diff
@@ -224,6 +249,16 @@ else if (PortletRequest.RENDER_PHASE.equals(lifecycle)) {
 			httpServletRequest, portlet, layout, lifecycle, copy);
 	}
 
+	private LiferayPortletURL _create(
+		HttpServletRequest httpServletRequest, Portlet portlet, Layout layout,
+		String lifecycle, MimeResponse.Copy copy, boolean isItNotificaitonPortlet) {
```

Comment:
Hi @kristofloneck could you please rename `isItNotificaitonPortlet` parameter to `notification`?